common/src/View/SmartSpawnflagsEditor.cpp

```diff
@@ -167,7 +167,7 @@ namespace TrenchBroom {
                     } else {
                         if (labels[i] != label) {
                             labels[i] = defaultLabels[i];
-                            tooltips[i] = "";
```

Comment:
`= "";` expresses the intent better imo, or `= wxString();` to avoid an implicit conversion?